lib/jira/connect.js

```diff
@@ -14,7 +14,7 @@ module.exports = async (req, res) => {
       name: `GitHub${isProd ? '' : (instance ? (` (${instance})`) : '')}`,
       description: 'Application for integrating with GitHub',
       key: `com.github.integration${instance ? `.${instance}` : ''}`,
-      baseUrl: `${isHttps ? 'https' : 'http'}://${req.get('host')}`,
+      baseUrl: process.env.APP_URL,
```

Comment:
Maybe we should check with Saquib that the `APP_URL` env var has the correct value in the Heroku environments before merging?

Reply:
`APP_URL` is needed for the app to run.  Currently staging is set to `github.stg.atlassian.com` while prod is currently set to their old domain `jira.github.com`, which we will soon ask to change to use our proxy.